spec/time.html

```diff
@@ -64,26 +64,19 @@ <h1>Temporal.Time.prototype</h1>
       <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
     </emu-clause>
 
-    <emu-clause>
-      <h1>Temporal.Time.fromString ( _isostring_ )</h1>
-      <p>
-        The `fromString` method takes one argument _isostring_.
-        The following steps are taken:
-      </p>
-      <emu-alg>
-        1. Let _isostring_ be ? ToString(_isostring_).
-        1. Return ? TimeFromString(_isostring_).
-      </emu-alg>
-    </emu-clause>
-
     <emu-clause>
       <h1>Temporal.Time.from ( _arg_ )</h1>
       <p>
         The `from` method takes one argument _arg_.
         The following steps are taken:
       </p>
       <emu-alg>
-        1. Return ? ToTime(_arg_).
+        1. If Type(_arg_) is Object, then
+          1. Return ? ToTime(_arg_).
+        1. If Type(_arg_) is String, then
+          1. Let _isostring_ be ? ToString(_arg_).
+          1. Return ? TimeFromString(_isostring_).
+        1. Throw a *TypeError* exception.
```

Comment:
Revert these changes.

Reply:
Again, this is what accurately coincides with the polyfill, but #269 will fix everything.